cms/static/sass/views/_settings.scss

```diff
@@ -271,6 +271,17 @@
             display: inline-block;
           }
         }
+
```

Comment:
realized it might be easier to add quick suggestions here: 

// UI: course pacing options
.group-settings.pacing {
    .list-input {
        @include margin-top($baseline/2); 
        background-color: $gray-l4; 
        border-radius: 3px;
        padding: ($baseline/2); 
    }

     .field {
         @include margin(0, 0, $baseline, 0);
      }

}

Outcome: 
![image](https://cloud.githubusercontent.com/assets/2023680/10770307/f461abd4-7cc0-11e5-9543-4a5430955f2f.png)